bin/cheritest/cheritest_vm.c

```diff
@@ -191,13 +191,11 @@ cheritest_vm_shm_open_anon_unix_surprise(const struct cheri_test *ctp __unused)
 		map = CHERITEST_CHECK_SYSCALL(mmap(NULL, getpagesize(),
 						PROT_READ, MAP_SHARED, fd,
 						0));
-		c = *map;
+		crx = *map;
 
-		fprintf(stderr, "rx cap: v:%lu b:%016jx l:%016zx o:%jx\n",
-			(unsigned long)cheri_gettag(c), cheri_getbase(c),
-			cheri_getlen(c), cheri_getoffset(c));
+		CHERI_FPRINT_PTR(stderr, crx);
```

Comment:
I'd prefer to use `_CHERI_PRINTF_CAP_FMT` and `_CHERI_PRINTF_CAP_ARG` rather than relying on the non-obvious behaviour of `CHERI_FPRINT_PTR` using the stringified expression as a prefix. That also means you don't need to rename the variables.